fearless/Common/Services/WebSocketService/StorageSubscription/ElectionStatusSubscription.swift

```diff
@@ -0,0 +1,140 @@
+import Foundation
+import RobinHood
+import FearlessUtils
+
+final class ElectionStatusSubscription: WebSocketSubscribing {
+    let childSubscriptionFactory: ChildSubscriptionFactoryProtocol
+    let runtimeService: RuntimeCodingServiceProtocol
+    let operationManager: OperationManagerProtocol
+    let engine: JSONRPCEngine
+    let logger: LoggerProtocol?
+
+    private let mutex = NSLock()
```

Comment:
To be more consistent I would suggest to call all locks the same way, either mutex or lock

Reply:
Thanks! Will fix in the separate PR